h/resmgr.h

```diff
@@ -40,69 +54,57 @@ ZOWE_PRAGMA_PACK_RESET
 #define resmgrDeleteAddressSpaceResourceManager RMGRDARM
 #endif
 
-/*****************************************************************************
-* Add task resource manager.
-*
-* Parameters:
-*   userRoutine               - user routine to be called when resource manager
-*                               is invoked
-*   userData                  - user data passed to user routine
-*   handle                    - handle containing resource manager information
-*   managerRC                 - resource manager return code
-*
-* Return value:
-*   RC_RESMGR_OK is returned when resource manager has been added.
-*   RC_RESMGR_SERVICE_FAILED is returned when an error occurred (see managerRC
-*   for details).
-*****************************************************************************/
+/**
+ * @brief Adds a task resource manager.
```

Comment:
Should we code @brief commands? Doxygen can be configured to implicitly handle brief comments using the JAVADOC_AUTOBRIEF option.

Reply:
Not sure. I've looked at existing projects and many use `@brief` explicitly.

Reply:
I tend to stick to Java compatible doc comments. But it's not a big issue for me.